lib/azure/armrest/wrapper/availability_set.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_wrapper'
+
+module Azure
+  module Armrest
+    class AvailabilitySet < Azure::ArmRest::BaseWrapper
```

Comment:
The idea here was that we could add custom methods to each class as-needed. It's a lot easier to debug and maintain IMHO.